modules/landing-pages/module.php

```diff
@@ -104,10 +94,10 @@ private function add_submenu_page() {
 		// If there are no Landing Pages, show the "Create Your First Landing Page" page.
 		// If there are, show the pages table.
 		if ( ! empty( $posts->posts ) ) {
```

Comment:
Check with @iNewLegend if it can be handled like the custom-code.

Reply:
If you are talking about the `render_blank_state`, I used some of its markup which I exported to another function: 

https://github.com/elementor/elementor/blob/98bc3b6bdffdf223bbba5d95576e22916f1e8783/includes/template-library/sources/local.php#L1327

I did this because some of the markup is relevant to the Template Library only. This means that in the custom code there is some markup that is template library-related.